db/index.go

```diff
@@ -123,16 +123,24 @@ func (i *collectionBaseIndex) getDocFieldValue(doc *client.Document) ([][]byte,
 	result := make([][]byte, 0, len(i.fieldsDescs))
 	for iter := range i.fieldsDescs {
 		fieldVal, err := doc.GetValue(i.fieldsDescs[iter].Name)
+		isNil := false
 		if err != nil {
 			if errors.Is(err, client.ErrFieldNotExist) {
```

Comment:
thought: :sweat_smile: Out of scope here perhaps, but we should add a `TryGetValue` func to `client.Document`.

Reply:
I like the idea.